service/dingding/dingding.go

```diff
@@ -0,0 +1,44 @@
+package dingding
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/blinkbean/dingtalk"
+	"github.com/pkg/errors"
+)
+
+type Service struct {
+	config Config
+	client *dingtalk.DingTalk
+}
+
+type Config struct {
+	Token  string
+	Secret string
+}
+
+func New(cfg *Config) *Service {
+	dt := dingtalk.InitDingTalkWithSecret(cfg.Token, cfg.Secret)
+	s := Service{
+		config: *cfg,
+		client: dt,
+	}
+	return &s
+}
+
+// Send takes a message subject and a message content and sends them to all previously set users.
+func (s *Service) Send(ctx context.Context, subject, content string) error {
+	select {
+	case <-ctx.Done():
+		return ctx.Err()
+	default:
+		text := fmt.Sprintf("%s\n%s", subject, content)
```

Comment:
```suggestion
		text := subject + "\n" + content
```

Please use regular string concatenation instead of `fmt.Sprintf`.